Branch-TestBed/Branch-TestBed/Branch-TestBed-Info.plist

```diff
@@ -19,16 +19,20 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.1</string>
+	<string>3.7.0</string>
```

Comment:
Tests are not happy with this change
`    XCTAssertEqualObjects(application.displayVersionString,         @"1.1");`